plugins/outputs/http/http_test.go

```diff
@@ -0,0 +1,100 @@
+package http
```

Comment:
In the next review, I will write test code using httptest. 
Then I will apply for a review. 
Thanks for the good info.